### PR TITLE
Make Settings UI the default experience

### PR DESCRIPTION
**Pull request by carlos-zamora**
This makes the settings UI the default settings experience.

As shown below, the following bindings are now default:
- <kbd>ctrl+,</kbd> --> settings ui
- <kbd>ctrl+shift+,</kbd> --> settings.json
- <kbd>ctrl+alt+,</kbd> --> defaults.json

The dropdown settings button aligns with this heuristic:
- click --> settings ui
- shift+click --> settings.json
- alt+click --> defaults.json
- if alt and shift both pressed, open settings.json

#6800 - Settings UI Epic

### CODE REVIEW

File: src/cascadia/TerminalApp/TerminalPage.cpp [hunk 1]
```diff
@@ -983,12 +983,34 @@ namespace winrt::TerminalApp::implementation
...
+        const auto rShiftState{ window.GetKeyState(VirtualKey::RightShift) };
+        const auto shiftPressed{ WI_IsFlagSet(shiftState, CoreVirtualKeyStates::Down) ||
+                                 WI_IsFlagSet(lShiftState, CoreVirtualKeyStates::Down) ||
+                                 WI_IsFlagSet(rShiftState, CoreVirtualKeyStates::Down) };
+
+        auto target{ SettingsTarget::SettingsUI };
+        if (shiftPressed && altPressed)
+        {
+            target = SettingsTarget::AllFiles;
+        }
```
Comment (DHowett) [inline src/cascadia/TerminalApp/TerminalPage.cpp:1005]: don't think we need both right now. They can bind it if they want. :smile:

Comment (carlos-zamora) [reply src/cascadia/TerminalApp/TerminalPage.cpp:1005]: So shift+alt click --> settings.json? (prioritize settings.json over defaults.json?)

**Review by zadjii-msft — CHANGES_REQUESTED** [review]
I have opinions on this matter and it's 5pm on a friday

**Review by zadjii-msft — APPROVED** [review]
With #9253, I'm cool with this for preview

File: src/cascadia/TerminalSettingsModel/ActionArgs.h [hunk 2]
```diff
@@ -444,7 +444,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         OpenSettingsArgs() = default;
         OpenSettingsArgs(const SettingsTarget& target) :
             _Target{ target } {}
-        GETSET_PROPERTY(SettingsTarget, Target, SettingsTarget::SettingsFile);
+        GETSET_PROPERTY(SettingsTarget, Target, SettingsTarget::SettingsUI);
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsModel/ActionArgs.h:447]: last q: should we leave this the same and only update the `target` in defaults.json? I'd like to make sure we don't break people who added their own openSettings key bindings.

**Review by DHowett — CHANGES_REQUESTED** [review]
Qs

File: src/cascadia/TerminalSettingsModel/ActionArgs.h (hunk 2 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h:447]: You're right. But I don't like that you're right hahaha. Begrudgingly making the change :P

Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h]: On a more serious note, should we consider this a breaking change we want to commit to for v2? Or for stable when SUI goes to stable?

Comment (DHowett) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h]: It costs us nothing to let users have their old bindings work the way they always have, right? Here's my thinking. If a user _bound settings to open_, IN their JSON file, that's probably what they are expecting.

**Review by DHowett — APPROVED** [review]

File: src/cascadia/TerminalSettingsModel/ActionArgs.h (hunk 2 above)
Comment (carlos-zamora) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h]: That's fair. And when the actual Actions page goes in, the user will probably have to explicitly choose what they want to open. So this seems fine :)

Comment (cinnamon-msft) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h]: Gah I hate to come in here and disagree after this has merged, but I think moving it to `SettingsTarget::SettingsUI` makes sense. I know we're going to be breaking people now, but a year from now I feel like we would have wished we moved the target to the settings UI.

Comment (DHowett) [reply src/cascadia/TerminalSettingsModel/ActionArgs.h]: We can make the decision to break peoples' settings in the future.

**Pull request merged**